src/app/config/Myservices.js

```diff
@@ -2,23 +2,25 @@ import { URL } from "./URL";
 import axios from "axios";
 
 export function registerationGet(email, password) {
-  return axios.get(`${URL}/Registration?email=${email}&password=${password}`);
+  return axios.get(
+    `${URL}/api/registeration?email=${email}&password=${password}`
```

Comment:
change url with host

Reply:
host variable is already used for another link.